integration-tests/src/test/java/com/amazonaws/services/schemaregistry/integrationtests/kafka/GlueSchemaRegistryKafkaIntegrationTest.java

```diff
@@ -194,12 +209,15 @@ public void testProduceConsumeWithSchemaRegistry(final DataFormat dataFormat,
         List<ProducerRecord<String, Object>> producerRecords =
                 kafkaHelper.doProduceRecords(producerProperties, records);
 
-        ConsumerProperties consumerProperties = ConsumerProperties.builder()
-                .topicName(topic)
-                .avroRecordType(recordType.name()) // Only required for the case of AVRO
-                .build();
+        ConsumerProperties.ConsumerPropertiesBuilder consumerPropertiesBuilder = ConsumerProperties.builder().topicName(topic);
+        String recordType = getRecordType(dataFormat, avroRecordType);
+        if (dataFormat.equals(DataFormat.PROTOBUF)) {
+            consumerPropertiesBuilder.protobufMessageType(recordType);
```

Comment:
Why can't we directly set the `protobufMessageType` to `DYNAMIC_MESSAGE` here? `getRecordType` is using the same the conditional check.